### PR TITLE
PR Template Updates

### PR DESCRIPTION
**Pull request by danielwheeler1987**
### Stakeholder Overview _[(learn more)](https://app.getguru.com/card/TGyLkrnc/Pull-Review-Stakeholder-Overview)_

This PR updates a few things:

1. It updates the `CODEOWNERS` file to designate Tailor Made & Premium Blend teams as owners of the PR template. This is to aid with team-wide communication / change management, especially after significant improvements like adding the Risk Assessment section last year. 
2. It introduces a new Data & Analytics Dependencies section to the PR template, to address issues identified during Accounts Verification conversions.
3. Changes some of the risks in the risk assessment to use the warning (⚠️) emoji instead of the green check (✅), for  a clearer at-a-glance assessment of a PR's risks. See below as an example for this PR.

### Risk Estimate _[(learn more)](https://app.getguru.com/card/iMnRRRjT/Pull-Request-Risk-Estimate)_

No Risk! This is just a PR Template & CODEOWNERS update.

- ⚠️ Not hidden by feature flag
- ✅ Negligible risk!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,3 @@
 * @customink/atlas
+
+.github/pull_request_template.md @customink/tailor-made @customink/premium-blend
```
Comment (invacuo) [inline .github/CODEOWNERS]: Nit: missing a new line at the end of the file.

**Review by invacuo — APPROVED** [review]

**Review by chmurph2 — COMMENTED** [review]
Can I take this opportunity to change this:

- ✅ Big/complex change
- ✅ Big splash zone
- ✅ High stakes if errors occur
- ✅ Low confidence
- ✅ Not hidden by feature flag
- ✅ Negligible risk!

to be this?

- ⚠️ Big/complex change
- ⚠️ Big splash zone
- ⚠️ High stakes if errors occur
- ⚠️ Low confidence
- ⚠️ Not hidden by feature flag
- ✅ Negligible risk!

I continue to find the ✅s for those bullets that amount to actual warnings to be misleading. I think that, at a glance, we should know this PR has some things to consider if there are any ⚠️s. 

I'm happy to open a separate PR for this too.

**Comment by danielwheeler1987** [comment]
@chmurph2 I'm fine with you making that change in this PR. We will want to provide insight to the teams this change is coming.

**Review by chmurph2 — APPROVED** [review]
Note that while the raw markdown syntax doesn't render the warning emoji, GitHub's rich text _does_ render it as an emoji:

CleanShot 2024-04-04 at 11 28 34@2x

CleanShot 2024-04-04 at 11 28 16@2x

**Review by a-maas — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -69,6 +69,11 @@ Common, optional tasks are included here in case you forgot something important.
 - Are there any new queries in your change set that might require new indexes?
 - Do any new queries require time-boxing to avoid table-scans when the data grows?
 
+
+##### Data & Analytics Dependencies
+- For customer-facing applications, are there any GA tagging changes that are needed or created by this change? If so, have those been discussed with D&A?
+- Has Data Engineering been notified of any schema changes?
```
Comment (mpeteuil) [inline .github/pull_request_template.md:75]: This should mainly be for modifications of existing schema attributes. Additions shouldn't matter. I don't know the details of what was discussed previously around this though.

Comment (danielwheeler1987) [reply .github/pull_request_template.md:75]: "Has Data Engineering been notified of any schema changes?" Should cover all the basis  here.

Comment (Shimmi) [reply .github/pull_request_template.md:75]: > Has Data Engineering been notified of any schema changes

Not sure this is the best wording or requirement. We do not do that unless it results in an impact on Datafuse configuration. Even if it does, we tend to prepare a PR to change that and run that true Data team.

**Comment by Shimmi** [comment]
> It introduces a new Data & Analytics Dependencies section to the PR template, to address issues identified during Accounts Verification conversions.
>
> _For customer-facing applications, are there any GA tagging changes that are needed or created by this change? If so, have those been discussed with D&A?_

I think, if this section had been there back then, we would still have answered _"Yes, there are"_ and _"Yes, we discussed it"_ to it...

Even tho this section may make people think about it for a second or two, I might put more focus on Why and What to do. Thinking out loud:
* Is that sufficient if the engineer simply answers _Yes and Yes_? Maybe we'd like engineers to describe the GA changes as well?
* If the answer is _Yes and No_, is it a necessarily a bad thing? One may introduce a new GA event that does not need to be consulted...

**Pull request merged**